### PR TITLE
Fix/trlst 233 tr fixes

### PR DESCRIPTION
**Pull request by davecharles**
Fixes:
- render parties view correctly for third party
- javascript confirmation box

### CODE REVIEW

File: trade_remedies_caseworker/cases/views.py [hunk 1]
```diff
@@ -1778,6 +1779,15 @@ def get(self, request, case_id, organisation_id, *args, **kwargs):
                 third_party_contacts = self.get_third_party_contacts(
                     org_id, org_submission_idx, all_case_invites
                 )
+            # `contacts` may also contain on-boarded third-party contacts that
+            # have a user, so wee need to prune these out.
```
Comment (rossmiller) [inline trade_remedies_caseworker/cases/views.py:1783]: "wee"

**Review by rossmiller — CHANGES_REQUESTED** [review]
One type

File: trade_remedies_caseworker/cases/views.py (hunk 1 above)
Comment (davecharles) [reply trade_remedies_caseworker/cases/views.py:1783]: Part of me wishes that was intentional 😂

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**